release_notes.md

```diff
@@ -4,6 +4,7 @@
 -->
 - Update Python Worker to 1.1.5 [Release Note](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.5)
 - Update Python Library to 1.3.1 [Release Note](https://github.com/Azure/azure-functions-python-library/releases/tag/1.3.1)
+- [CustomHandler][Breaking] Outputs assigned to output binding are expected at least have an empty JObject. It now throws an exception if it doesn't.
```

Comment:
change to 
If `enableForwardingHttpRequest` is false, http output binding response is expected to be a valid Json object with following optional fields :
```json
{
"statusCode" : "",
"status" : "",
"body": "",
"headers" : {}
}
```
Excpetion is thrown if HttpOutputBindingResponse is not valid Json